kbase-extension/static/kbase/config/feature-config.json

```diff
@@ -1,5 +1,5 @@
 {
-    "stagingDataViewer": false,
+    "stagingDataViewer": true,
```

Comment:
Is this okay?  For next I'm guessing yes, but I want to be sure.  What about appdev and prod?